### PR TITLE
[DNC] Stable release 03_15_21

### PR DESCRIPTION
**Pull request by tamilmani1989**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [x] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  -  [ ] Ensure to copy the existing version into new directory structure for first commit (including refactoring) and then push new changes including version updates in separate commits. This is required to review the changes efficiently.
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/dnc/resource-manager/readme.md [hunk 1]
```diff
@@ -97,22 +111,36 @@ output-folder: $(azure-libraries-for-java-folder)/azure-mgmt-DelegatedNetwork
...
+These settings apply only when `--tag=package-2020-08-08-preview --java` is specified on the command line.
+Please also specify `--azure-libraries-for-java=<path to the root directory of your azure-sdk-for-java clone>`.
+
+```yaml $(tag) == 'package-2020-08-08-preview' && $(java) && $(multiapi)
+java:
+  namespace: com.microsoft.azure.management.DelegatedNetwork.v2020_08_08_preview
+  output-folder: $(azure-libraries-for-java-folder)/sdk/DelegatedNetwork/preview/mgmt-v2020_08_08_preview
+  regenerate-manager: true
+  generate-interface: true
+```
```
Comment (ArcturusZhang) [inline specification/dnc/resource-manager/readme.md:142]: Could we extract the java configurations to its own file?

Comment (tamilmani1989) [reply specification/dnc/resource-manager/readme.md:142]: done

File: specification/dnc/resource-manager/Microsoft.DelegatedNetwork/stable/2021-03-15/orchestrators.json [hunk 2]
```diff
@@ -0,0 +1,504 @@
...
+        },
+        "clusterRootCA": {
+          "description": "RootCA certificate of kubernetes cluster base64 encoded",
+          "type": "string"
+        },
+        "apiServerEndpoint": {
+          "description": "K8s APIServer url. Either one of apiServerEndpoint or privateLinkResourceId can be specified",
+          "type": "string"
+        },
+        "privateLinkResourceId": {
```
Comment (pilor) [inline specification/dnc/resource-manager/Microsoft.DelegatedNetwork/stable/2021-03-15/orchestrators.json:451]: Incorporating private links typically takes more orchestration than a single property (i.e. requests + approvals).  Can you explain where in the private endpoints/private links scenario this falls according to Networkings docs:  https://microsoft.sharepoint.com/teams/vnet/SitePages/Private-Endpoints.aspx

**Review by pilor — CHANGES_REQUESTED** [review]

File: specification/dnc/resource-manager/Microsoft.DelegatedNetwork/stable/2021-03-15/orchestrators.json (hunk 2 above)
Comment (tamilmani1989) [reply specification/dnc/resource-manager/Microsoft.DelegatedNetwork/stable/2021-03-15/orchestrators.json:451]: @pilor  The need for private link is customer running their apiserver in private ip and our control plane need a way to access the server from outside. So our RP will create Private endpoint as part of orchestrator resource creation using privatelink resource id. Can directly access private endpoint ip to reach the server. No need for dns integration

 Yes customer will manually approve our request once  orchestrators resource is created.

Comment (pilor) [reply specification/dnc/resource-manager/Microsoft.DelegatedNetwork/stable/2021-03-15/orchestrators.json:451]: Ok, ensure you are following networking guidelines from that link.  I am not a private link expert so I can't confirm that you are following all the necessary steps

**Review by pilor — APPROVED** [review]

**Review by ArcturusZhang — APPROVED** [review]

**Pull request merged**